tests/ui_tools/test_boxes.py

```diff
@@ -54,6 +56,14 @@ def test_init(self, write_box):
         assert write_box.model == self.view.model
         assert write_box.view == self.view
         assert write_box.msg_edit_state is None
+        assert write_box.msg_body_edit_enabled is True
+        assert write_box.stream_id is None
+        assert write_box.recipient_user_ids == []
+        assert write_box.typing_recipient_user_ids == []
+        assert write_box.to_write_box is None
+        assert isinstance(write_box.send_next_typing_update, datetime.datetime)
+        assert isinstance(write_box.last_key_update, datetime.datetime)
```

Comment:
This isn't the best way to test these initializations, but I'm unable to patch the `datetime.now` method (I'm not sure if we can), and this seemed like a reasonable alternative.

Reply:
This is a good improvement for now - I think we'd need to adjust how we handle datetime otherwise.